tests/test_api.py

```diff
@@ -62,15 +63,16 @@ def test_veracode_xml_api_version(self):
     ## VeracodeXMLAPI app_id property
     def test_veracode_xml_api_app_id(self):
         """
-        Test the VeracodeXMLAPI app_id property
+        Test the VeracodeXMLAPI app_name property
         """
-        veracode_xml_api = VeracodeXMLAPI()
+        with patch("veracode.api.get_app_id", return_value=0):
+            veracode_xml_api = VeracodeXMLAPI("TestApp")
```

Comment:
Where possible, be specific
```suggestion
            veracode_xml_api = VeracodeXMLAPI(app_name="TestApp")
```